tests/bsim/bluetooth/host/gatt/ccc_store/testcase.yaml

```
@@ -15,3 +15,8 @@ tests:
       bsim_exe_name: tests_bsim_bluetooth_host_gatt_ccc_store_overlay-no_store_on_write_conf
     extra_args:
       EXTRA_CONF_FILE=overlay-no_store_on_write.conf
+  bluetooth.host.gatt.ccc_store_3:
+    harness_config:
+      bsim_exe_name: tests_bsim_bluetooth_host_gatt_ccc_store_overlay-no_long_wq_conf
+    extra_args:
+      EXTRA_CONF_FILE=overlay-no_long_wq.conf
```

Just FYI, you could also have done the following (I think)

```suggestion
    extra_args:
      - CONFIG_BT_LONG_WQ=n
```

If you didn't want that extra config file

---

I am aware that we can use `extra_configs` here. I aligned my PR with the existing approach (there are already Kconfig overlay files that modify value of a single Kconfig option and introduce a comment).